boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -159,8 +159,8 @@
 &flexcan2 {
 	pinctrl-0 = <&flexcan2_default>;
 	pinctrl-names = "default";
-	bus-speed = <125000>;
-	bus-speed-data = <1000000>;
+	bitrate = <125000>;
+	bitrate-data = <1000000>;
```

Comment:
This is wrong, they should be removed instead. See https://github.com/zephyrproject-rtos/zephyr/pull/73806

Reply:
Ah, gotcha, I was just looking at the DTS changes on it and going off the reported error. Ok, since you have that one in I'll close this one.